toolkits/springernature/packages/springernature-header/scss/50-components/enhanced.scss

```diff
@@ -22,6 +24,7 @@
 
 .c-header__site-logo {
 	padding-right: 20px;
+	border-right: 2px solid #222; // For SNAPP logo border-right
```

Comment:
See rule `.c-header__site-logo.with-journal-logo`, line 36
If there is a journal logo, add class `with-journal-logo` to element with class `c-header__site-logo`